recipes/opencv/2.x/conanfile.py

```diff
@@ -0,0 +1,248 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
```

Comment:
```suggestion
import os

required_conan_version = ">=1.29.1"

```

Because of `tools.remove_files_by_mask()` used in `package()`: https://github.com/conan-io/conan/releases/tag/1.29.1

Reply:
Done